src/qt/guiconstants.h

```diff
@@ -26,8 +26,8 @@ static const bool DEFAULT_SPLASHSCREEN = true;
 #define RGB_HALF 0x7f7f7f
 /** Path to the icon resource folder */
 #define ICONS_PATH ":icons/"
-/** Path to the movies resource folder */
-#define MOVIES_PATH ":movies/"
+/** Path to the animation resource folder */
+#define MOVIES_PATH ":animation/"
```

Comment:
gui-14: maybe `s/MOVIES_PATH/ANIMATION_PATH/` for consistency (also in `src/qt/bitcoingui.cpp`)

Reply:
dropping it was not smth I suggested 😅 but it's used in one single place only so I guess it's ok.

Reply:
I thought to drop `ICONS_PATH` also but they used twice... so... I keep them